### ISSUE TITLE
Edit Windows Guide

### ISSUE THREAD

**Issue by adammichaelwood** [opendatakit/docs#159]
@nadaa has recently completed a guide to building the docs in Windows and working locally with Cygwin. 

The content of this guide is very good, but there are two problems:

 1.  @nadaa, like many of us, is relatively new to this kind of technical writing, so there are style issues and rewrites that need to be done
 2. I do not have a Windows machine, so there is no way for me to check the accuracy or usefulness of the document.

In past PRs, I have done several rounds of back-and-forth editorial work in the comments, waiting until the original committer had "gotten everything right" before merging. 

For this document, I would like to **try an experiment.**

This may not work, but I'd like to try it out.

I am looking for **one person who has a Windows machine** to:

 - follow the guide
 - correct or clarify any content
 - provide editorial corrections
     - this is more than line edits. Editorial work means you have an opinion about how best to communicate an idea, and you are empowered to add, edit, delete, or rearrange existing content.
 - submit a PR (referencing this issue) with your proposed changes.

Once the PR is submitted, I would like @nadaa to complete a Review of that PR.
 

Please ask any questions you have here before diving in. The first person to claim it will do the PR. There will be other opportunities to do this, so don't worry if you can't do this one.

**Comment by Komal-Agarwal96** [comment]
@adammichaelwood  Sir, can I work on this?

**Comment by Divya063** [comment]
@adammichaelwood I have installed the development environment using command prompt only, on windows.Installation steps for the development environment can be provided as an alternative if someone doesn't want to install Cygwin

**Comment by adammichaelwood** [comment]
@Komal-Agarwal96  Yes.

@Divya063 You mentioned you are on Windows 10. Can I assume you used the Linux subsystem?

**Comment by Divya063** [comment]
@adammichaelwood No, I didn't use Linux subsystem.

**Comment by adammichaelwood** [comment]
@Divya063 let's continue this conversation on #161

**Comment by adammichaelwood** [comment]
(Deleted off-topic comments, for clarity)

**Comment by adammichaelwood** [comment]
@Komal-Agarwal96  

You are working on this, yes?

**Comment by Komal-Agarwal96** [comment]
@adammichaelwood  Since cygwin.rst is going to be re-written anyway as you mentioned in #96 should I continue with this or not?

**Comment by adammichaelwood** [comment]
> Since cygwin.rst is going to be re-written anyway

Rewriting `cygwin.rst` is this project .

**Comment by Komal-Agarwal96** [comment]
Okay, then I am on it.

**Comment by adammichaelwood** [comment]
Are you able to work on an edit/rewrite? Please read the description above.

**Comment by Komal-Agarwal96** [comment]
Yes, sir. I had started but got confused because of the comment in #96. Will correct/clarify content, rewrite and make line edits as described in the description above. Thanks

**Comment by nadaa** [comment]
@adammichaelwood, shall I close this issue?
This idea was great, but it didn't move forward. It stopped in the middle.
@Komal-Agarwal96 suggested some good points while trying to follow the steps, but she added new stuff without explanation. I couldn't approve what she wrote if it doesn't make sense.

**Comment by adammichaelwood** [comment]
Why don't you see if there's any good suggestions/edits that are worth making into your original, and submit a PR that you feel improves your doc. Have that PR close this issue.

**Comment by ManognaMutyala** [comment]
Can I add few more images to the documentation regarding this?

**Comment by adammichaelwood** [comment]
Part of #402
